functions/cond.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any, Callable
+
+from .curry import curry
+from .is_truthy import is_truthy
+
+
+@curry
```

Comment:
No `@curry` is required. because the function is returned regardless of the number of parameters